src/ripple/app/main/Application.cpp

```diff
@@ -1714,7 +1713,12 @@ ApplicationImp::run()
     JLOG(m_journal.info()) << "Received shutdown request";
     stop(m_journal);
     if (config_->reporting())
```

Comment:
I want to make the calls to start and stop symmetrical so that the change to RAII will be straightforward and easy to review. I want to give `stop` methods the pre-condition that the object is started, just like `std::thread::join` assumes the thread is active.